packages/user/src/User/Login.purs

```diff
@@ -34,11 +36,14 @@ import KSF.ValidatableForm as Form
 import React.Basic (JSX)
 import React.Basic.Classic (make)
 import React.Basic.Classic as React
+import React.Basic.Classic as React.Classic
```

Comment:
`React.Basic.Classic` is imported twice as `React` and `React.Classic`. Surely once is enough? And later `React.Basic.Hooks` gets imported as `React`, too.